async-nats/tests/client_tests.rs

```diff
@@ -15,6 +15,8 @@ mod nats_server;
 
 mod client {
 
+    use std::{collections::HashMap, time::Duration};
```

Comment:
Clippy says HashMap is never used here.

```suggestion
    use std::{time::Duration};
```